diesel/src/r2d2.rs

```diff
@@ -31,11 +31,9 @@ use crate::sql_types::HasSqlType;
 #[derive(Debug, Clone)]
 pub struct ConnectionManager<T> {
     database_url: String,
-    _marker: PhantomData<T>,
+    _marker: PhantomData<fn() -> T>,
```

Comment:
I think, you should leave comment from PR description that explains, why not just `PhantomData<T>` is used